--- /etc/neutron/ml2_conf_opencontrail.ini.orig	2018-05-03 17:22:32.223394206 +0000
+++ /etc/neutron/ml2_conf_opencontrail.ini	2018-05-03 17:22:48.103627692 +0000
@@ -2,6 +2,8 @@
 [APISERVER]
 api_server_port = 8082
 api_server_ip = 192.168.100.6
+use_ssl = False
+auth_token_url = http://192.168.100.6/identity_admin
 # (StrOpt) IP address for OpenContrail API Server
 #          This is a required field. If not set, all communications to
 #          OpenContrail API Server will fail.
