--- opencontrail_sg_callback.py.orig	2018-05-05 21:42:33.314909662 +0000
+++ opencontrail_sg_callback.py	2018-05-05 21:42:49.148125723 +0000
@@ -14,9 +14,9 @@
 #
 
 from neutron._i18n import _LE
-from neutron_lib.callbacks import events
-from neutron_lib.callbacks import registry
-from neutron_lib.callbacks import resources
+from neutron.callbacks import events
+from neutron.callbacks import registry
+from neutron.callbacks import resources
 from oslo_log import helpers as log_helpers
 from oslo_log import log as logging
 from oslo_utils import excutils
